### DISCUSSION TITLE
How to edit the Community page?

### DISCUSSION THREAD

**Discussion by DanMax03** [cpeditor/cpeditor.github.io#180] [Q&A] [answered]
I'm trying to make Russian language for the Community page https://cpeditor.org/ru/community/ and looking in content/community/ for the file to edit, but there's nothing except the header in _index.ru.md. So where is it?

**Comment by ouuan** [comment, marked as answer]
It's not translatable as for now: https://github.com/cpeditor/cpeditor.github.io/blob/f7371847179bebfbea297da7dfa07a756c11d8f6/config.toml#L159-L176

We need to edit the theme in order to make it translatable. You can skip it first.
